vit-servicing-station-tests/src/tests/bootstrap/parameters.rs

```diff
@@ -84,3 +88,23 @@ pub fn in_settings_file_with_malformed_path() {
         .failure()
         .code(ApplicationExitCode::LoadSettingsError as i32);
 }
+
+#[test]
+pub fn db_url_and_block0_replaced() -> Result<(), Box<dyn std::error::Error>> {
+    let temp_dir = TempDir::new().unwrap();
+
+    let (token, hash) = data::token();
+
+    let db_path = DbBuilder::new()
+        .with_token(token)
+        .with_proposals(data::proposals())
+        .build(&temp_dir)?;
+
+    let server = ServerBootstrapper::new()
+        .with_block0_path(db_path.to_str().unwrap())
+        .with_db_path(BLOCK0_BIN)
+        .start()?;
```

Comment:
The server will not be able to bootstrap now and will return the DB connection trouble exit code.